utils/custom_load.py

```diff
@@ -473,13 +498,13 @@ def JsonVisualizer(self,dir_path = None):
         ext = 'json'
         return self._Visualizer(ext,dir_path)
     
-    def SavetoLas(self, results, dir_path = "results/"): 
-
+    def SavetoLas(self, results, dir_path = None): 
+                
         print("Saving point cloud in LAS format with segementation results...")
         if dir_path is None:
-            dir_path = os.path.join(self.dir, "/las")
-            print(f"Folder {dir_path} created in current directory.")
-        elif not os.path.exists(dir_path):
+            dir_path = os.path.join(self.dir, "las")
```

Comment:
need to have forward slash after las? ` dir_path = os.path.join(self.dir, "las/")` ?

Reply:
no need. I tried to print them, the os.path.join will automatically provide a forward slash. You can check the object instances at the very start. Try and print them out to see them as well.

Reply:
Got it. I will merget.